cilium/cmd/encrypt_status.go

```diff
@@ -0,0 +1,137 @@
+// Copyright 2021 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"fmt"
+	"os/exec"
+	"reflect"
+	"regexp"
+	"strings"
+
+	"github.com/cilium/cilium/pkg/command"
+	"github.com/cilium/cilium/pkg/common"
+	"github.com/cilium/cilium/pkg/option"
+	"github.com/prometheus/procfs"
+	"github.com/vishvananda/netlink"
+
+	"github.com/spf13/cobra"
+)
+
+const (
+	// EncryptionModeDisabled captures enum value "Disabled"
+	EncryptionModeDisabled string = "Disabled"
+
+	// EncryptionModeIPsec captures enum value "IPsec"
+	EncryptionModeIPsec string = "IPsec"
+
+	// EncryptionModeWireguard captures enum value "Wireguard"
+	EncryptionModeWireguard string = "Wireguard"
```

Comment:
Wireguard is actually always spelled with a capital W and capital G.
```suggestion
	EncryptionModeWireguard string = "WireGuard"
```

Reply:
HealthAPI returns enum value as 'wiregurad' so i just used the same.

Reply:
Can we change it to return "WireGuard"? That would be best.

Reply:
Yes sure so should i make the changes in the API or just the CLI output ?

Reply:
I raised that issue with people more familiar than me with the API and its behavior. We can't change it in the API because it would be a breaking change (it's case sensitive). It probably doesn't matter much here so let's leave it at `Wireguard`. If someone complains about the case, they can always submit a PR to add some ugly code for `Wireguard` -> `WireGuard` before display.

Reply:
Also, consider pulling in the `models.EncryptionStatusMode` constants defined in the API, instead of re-declaring them:

https://github.com/cilium/cilium/blob/173159664f149e0c6902fee645f0b2204c804f67/api/v1/models/encryption_status.go#L70-L77